packages/eslint-plugin-pf-codemods/src/rules/helpers/pfPackageMatches.ts

```diff
@@ -16,9 +16,11 @@ export function pfPackageMatches(
       parts[1] +
       "(/dist/(esm|js|dynamic))?" +
       (parts[2] ? "/" + parts[2] : "") +
-      `(/(components|helpers${
-        parts[1] === "react-icons" ? "|icons" : ""
-      })/.*)?$`
+      (parts[1] === "react-component-groups"
+        ? `(/.*)?$`
+        : `(/(components|helpers${
+            parts[1] === "react-icons" ? "|icons" : ""
+          })/.*)?$`)
```

Comment:
Copy of what was done in #710 to minimize conflicts, though I really feel like this helper should be cleaned up as this dynamic regex is getting _really_ gnarly.